src/utils/stripLocationInfo.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Returns a copied version of the node recieved without location attributes.
+ * @param  {Object}  node
+ * @return {Object}  node
+ */
+export default function stripLocationInfo(node) {
```

Comment:
I use it to copy the superClass node without the location info. I don't actually use it for comparisons.